libs/ui/.storybook/preview.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { DecoratorFunction, Parameters } from '@storybook/types';
+
+import { AppBase } from '../src';
+
+export const parameters: Parameters = {
+  actions: { argTypesRegex: '^on[A-Z].*' },
+  controls: {
+    matchers: {
+      color: /(background|color)$/i,
+      date: /Date$/,
+    },
+  },
+};
```

Comment:
Ah, this was boilerplate generated by storybook - it basically matches any prop name that matches the pattern on the right to the control type on the left (so a `textColor` prop will cause storybook to render a visual color selector control instead of just a text field for the color, and `startDate` will render a calendar control)

Glad you asked - makes me realize a comment will be useful here